app/controlplane/api/controlplane/v1/response_messages.proto

```diff
@@ -28,6 +28,7 @@ message WorkflowItem {
   string id = 1;
   string name = 2;
   string project = 3;
```

Comment:
and what's project?

Reply:
It's the project name.

Reply:
and why do we need the ID now?

Reply:
Yes, it's a feature for RBAC. Workflows are already filtered by visibility, but the ID would be useful to check exact permissions (read or write) on the workflow.